extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -530,6 +580,14 @@ public static Token fromAudience(String... audience) {
         @ConfigItem
         public Optional<String> principalClaim = Optional.empty();
 
+        /**
+         * The time, in seconds, that tokens should be silently refreshed so that tokens with no active session at the
```

Comment:
@pedroigor As mentioned earlier, IMHO it would be better if we drop `silent` and any reference to the logout. We just say something like "Refresh token timeout in seconds. If enabled, the expired ID token will be refreshed if the difference between the current time and the issued at time is less than this timeout" (but in a better worded way :-) ). And may be add " and will invalidate the old sessions at the RP and OP sites." or similar as you prefer